src/elements/list/src/index.tsx

```diff
@@ -89,6 +109,8 @@ export const ListItem: React.FC<ListItemProps> = ({
 
   return (
     <li sx={sx} {...props}>
+      {variant === 'pros' && <Icon color="green" glyph="tick" size={24} />}
+      {variant === 'cons' && <Icon color="red" glyph="cross" size={24} />}
```

Comment:
I'm not sure how the Icon maps the `color` to the theme, but we should probably have it come from there, e.g.

```
sx={{
  fill: 'elements.icon.positive'
}}
```